netsim/ansible/templates/gateway/cumulus_nvue.j2

```diff
@@ -36,7 +36,7 @@
                 address:
 {%     if 'ipv6' in intf.gateway %}
 {# Per RFC9568 5.2.9, the link-local ipv6 address MUST be included first. Required for interop with EOS (for example) #}
-                  fe80::200:5eff:fe00:2{{ '%02x' | format(intf.gateway.vrrp.group) }}: {}
+                  {{ intf.gateway.vrrp.lla }}: {}
```

Comment:
There are also valid ipv4 link local addresses, perhaps ```ipv6_lla``` would be a better name for this attribute

Reply:
While you're technically correct, the IPv4 LLAs have no special meaning in VRRP.